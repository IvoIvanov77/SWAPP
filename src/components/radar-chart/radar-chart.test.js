import React from 'react';
import { shallow } from 'enzyme';
import RadarChart from './radar-chart.component';

it('should render RadarChart component', () => {
  expect(shallow(<RadarChart />)).toMatchSnapshot();
});
