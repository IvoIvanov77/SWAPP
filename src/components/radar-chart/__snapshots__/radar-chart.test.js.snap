// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Episode component should render Episode component 1`] = `ShallowWrapper {}`;
